src/components/ebay-menu/test/test.server.js

```diff
@@ -107,7 +108,19 @@ describe('menu', () => {
         const input = { label: '' };
         const $ = testUtils.getCheerio(context.render(input));
         expect($(labelSelector).length).to.equal(0);
-        expect($('.expand-btn__icon').length).to.equal(1);
+        expect($('svg.expand-btn__icon').length).to.equal(1);
+    });
+
+    test('renders with icon', context => {
+        const input = { icon: 'settings', iconTag: { renderBody: mock.iconRenderBody } };
+        const $ = testUtils.getCheerio(context.render(input));
+        expect($('div.expand-btn__icon').html()).to.equal('icon');
```

Comment:
This is breaking the CI since marko v4 is outputting some placeholder comments here. I think a good fix would be to change to `.text()` instead of `.html()`?

Reply:
Oops, thanks